docs/try/tutorials/geomodel/geomodel-basics.md

```diff
@@ -1,22 +1,76 @@
 
-## How to build and use GeoModel
-### GeoModelBuilder
+## Basic functionalities on GeoModel
+The most efficient way to build a GeoModel is to import its geometry and topology from a
+geomodeler. This tutorial present a simple code to load manipulate and export GeoModel
+
+### GeoModel I/O
+
+GeoModel store the geometry and the topology of a geological model. It can be imported from boundary 
+representation or volumetric representation file format.
+RINGMesh use a factory design pattern to allow an easy implementation of various Input/Output management. 
```

Comment:
management => file format ?

Reply:
I do not want to put an exautive list her because it will be painfull to maintain. We already have a webpag to do so.

Reply:
management is in french, in english it is managment but I think file format is better.